articles/communication-services/concepts/service-limits.md

```diff
@@ -80,17 +80,28 @@ If you have requirements that exceed the rate-limits, submit [a request to Azure
 For more information on the SMS SDK and service, see the [SMS SDK overview](./sms/sdk-features.md) page or the [SMS FAQ](./sms/sms-faq.md) page.
 
 ## Email
-Sending a high volume of messages has a set of limitations on the number of email messages you can send. If you hit these limits, your messages won't be queued to be sent. You can submit these requests again, once the Retry-After time expires.
+There is a limit on the number of email messages you can send. If you exceed the below limits on your subscription, your requests will be rejected. You can attempt these requests again, after the Retry-After time has passed. Please take the action and request to raise the sending volume limits.
```

Comment:
The final sentence is difficult to understand. Can you revise?

Reply:
Maybe: Please take necessary action and request to raise the sending volume limits if needed.